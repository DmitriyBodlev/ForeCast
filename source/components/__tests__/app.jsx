// Standalone runtime for Regenerator-compiled generator and async functions
import 'regenerator-runtime/runtime';

import React from 'react';
import { shallow } from 'enzyme';
import App from './../app';
import routes from '../../routes-config';

describe('source/app.jsx', () => {
  it('should render component with routes', () => {
    const wrapper = shallow(<App routes={routes} />);

    expect(wrapper).toMatchSnapshot();
  });
});
