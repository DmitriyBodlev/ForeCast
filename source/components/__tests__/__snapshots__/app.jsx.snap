// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`source/app.jsx should render component with routes 1`] = `
<div>
  <RouteWithSubRoutes
    component={[Function]}
    key="0"
    path="/login"
  />
  <RouteWithSubRoutes
    component={[Function]}
    key="1"
    path="/reset_password"
  />
  <RouteWithSubRoutes
    component={[Function]}
    key="2"
    path="/new_password"
  />
  <RouteWithSubRoutes
    exact={true}
    key="3"
    path="/"
    routes={
      Array [
        Object {
          "component": [Function],
          "path": "/",
        },
        Object {
          "component": [Function],
          "path": "/users/:guid/settings",
        },
        Object {
          "component": [Function],
          "path": "/users/list",
        },
        Object {
          "component": [Function],
          "path": "/users/new",
        },
        Object {
          "component": [Function],
          "path": "/enterprises/new",
        },
        Object {
          "component": [Function],
          "path": "/enterprises/list",
        },
        Object {
          "component": [Function],
          "path": "/enterprises/details/:guid",
        },
        Object {
          "component": [Function],
          "path": "/enterprises/:guid/edit",
        },
      ]
    }
  />
</div>
`;
